helios-ts/lib.ts

```diff
@@ -19,8 +20,15 @@ export class HeliosProvider {
     const consensusRpc = config.consensusRpc;
     const checkpoint = config.checkpoint;
     const network = config.network ?? Network.MAINNET;
+    const dbType = config.dbType ?? "local_storage";
```

Comment:
Lets make the default `localstorage` since this has the best compatibility across different wasm environments